packages/web/src/components/buy-audio-modal/components/AudioAmountPicker.tsx

```diff
@@ -105,17 +105,23 @@ export const AudioAmountPicker = ({
     }
   }, [isCustomAmountInputVisible, customAmountRef])
 
+  const id = useMemo(() => uniqueId(), [])
+
   return (
     <>
       {!isCustomAmountInputVisible ? (
         <AmountPreview amount={presetAmount} />
       ) : null}
       <RadioButtonGroup
+        aria-labelledby={`audioAmountPicker-label-${id}`}
```

Comment:
Just wondering if we need a random Id? Are there gonna be multiple rendered?

Reply:
Not currently, but better safe than sorry imo